monkestation/code/modules/mob/living/carbon/human/species_type/ipc.dm

```diff
@@ -194,23 +196,30 @@
 	H.dna.features["ipc_screen"] = "BSOD"
```

Comment:
would be nice if u cleaned up the single letter vars, but not a requirement

Reply:
Good point, may as well clean it up while I'm here. Done.